_includes/k8s-spec-generate.md

```diff
@@ -4,11 +4,40 @@ To generate the spec file for the 1.3 release, head on to [1.3 install page](htt
 
 Alternately, you can use curl to generate the spec as described in [Generating Portworx Kubernetes spec using curl](/scheduler/kubernetes/px-k8s-spec-curl.html).
 
->**Secure ETCD:**<br/> If using secure etcd provide "https" in the URL and make sure all the certificates are in the _/etc/pwx/_ directory on each host which is bind mounted inside PX container.
+#### Secure ETCD and Certificates
+If using secure etcd provide "https" in the URL and make sure all the certificates are in the _/etc/pwx/_ directory on each host which is bind mounted inside PX container.
 
+#### Using Secrets to Provision Certificates
+>**Note for ASG users:**<br/>When using PX with Auto Scaling Groups in AWS, you should use Secrets to have Kubernetes provision the ETCD certificates.  This way, the certificates will be provisioned on any new EC2 instance automatically.
```

Comment:
This isn't specific to ASG. But rather can be done in any k8s cluster. So we can reword this to

```
It is recommended to use Kubernetes Secrets to provide ETCD certificates to Portworx. This way, the certificates will be automatically mounted when new nodes join the cluster."
```